packages/core/src/internal/data-grid/data-grid-types.ts

```diff
@@ -313,6 +313,8 @@ export interface BaseGridCell {
 /** @category Cells */
 export interface LoadingCell extends BaseGridCell {
     readonly kind: GridCellKind.Loading;
+    readonly skeletonWidth?: number;
+    readonly skeletonVariability?: number;
```

Comment:
Think instead of `skeletonVariability`, `skeletonWidth` should either be a static `number` or a dynamic `(col: number, row: number) => number`, so it can left up to developer to be pseudo-random or deterministic.

Reply:
The callback seems not needed as you get that information inside `getCellContent`. The variability is just there to make it easier to achieve semi-organic skeletons without writing your own randomizer. I just didn't want it to be the obvious fallback that people use Math.random() which is rather slow.